charts/kubewarden-controller/README.md

```diff
@@ -82,6 +79,28 @@ or on [docs.crds.dev](https://doc.crds.dev/github.com/kubewarden/kubewarden-cont
 
 **Note well:** `ClusterAdmissionPolicy` resources are cluster-wide.
 
+### Deploy your Policy Server
+
+Before start deploying your `ClusterAdmissionPolicy` you need to have a Policy
+Server running where the policy will be run. You can deploy one like this:
+
+```bash
+kubectl apply -f - <<EOF
+---
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: PolicyServer
+metadata:
+  name: default
+spec:
+  image: ghcr.io/kubewarden/policy-server:v0.2.6
+  replicas: 1
+  serviceAccountName: sa
```

Comment:
user will get permission errors if serviceAccount `sa` doesn't exist or doesn't have enough permissions

Reply:
I would leave the default `PolicyServer` see [here](https://github.com/kubewarden/helm-charts/pull/65#issuecomment-1058993873)
or document how to create the serviceAccount if we really want to remove the default `PolicyServer`